data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,64 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#Historical Raptor by Player
+historical_raptor_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") %>%
+  mutate(
+    era = factor(c("historical"))
+  ) 
+historical_raptor_by_player[c("raptor_box_offense", "raptor_box_defense","raptor_box_total", 
+            "raptor_onoff_offense", "raptor_onoff_defense", "raptor_onoff_total")] <- NA
```

Comment:
Just so we can stay consistent in coding style throughout this package, could you do this with a `mutate`? Ex:
```
  mutate(
    raptor_box_offense = NA, 
    raptor_box_defense = NA, 
    ...
  )
```